pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -127,8 +130,16 @@ func (a *ofCTAction) natAction(isSNAT bool, ipRange *IPRange, portRange *PortRan
 
 	// ipRange should not be nil. The check here is for code safety.
 	if ipRange != nil {
-		action.SetRangeIPv4Min(ipRange.StartIP)
-		action.SetRangeIPv4Max(ipRange.EndIP)
+		if utilnet.IsIPv6(ipRange.StartIP) {
+			action.SetRangeIPv6Min(ipRange.StartIP)
+		} else {
+			action.SetRangeIPv4Min(ipRange.StartIP)
+		}
+		if utilnet.IsIPv6(ipRange.EndIP) {
```

Comment:
I think ipRange won't mix protocol family, so L139 can be after L134?

Reply:
Indeed